tests/ci/run_tests_with_sde.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+source tests/ci/common_posix_setup.sh
+
+echo "Testing AWS-LC in debug mode."
```

Comment:
Nit: change this message to mention that the SDE tests are being run

Reply:
Changed.